nextcloudappstore/core/models.py

```diff
@@ -106,6 +106,7 @@ class Meta:
         verbose_name = _('App Release')
         verbose_name_plural = _('App Releases')
         unique_together = (('app', 'version'),)
+        ordering = ['-version']
```

Comment:
Oh, that'd be great. We'd need to migrate incoming versions to a spec but that would make sense anyways. If we wanted to allow -beta suffixes we'd also just have to change the xsd which would be awesome :)